### ISSUE TITLE
GhostScript Font Sizes

### ISSUE THREAD

**Issue by abenson** [taviso/wpunix#25]
Is there a trick to getting the font sizes (fine, large, very large, etc) working with GhostScript?  I see 0.10 reverted the GhostScript for this issue, but even after removing/adding the printer for it, [the font sizes still won't change](https://i.imgur.com/MFxxBVF.png).

Switching to a different printer driver, like HP Color LaserJet PS, does print correctly, but I don't want to do that since it doesn't support modern characters, like '€'.

Side note: I really appreciate the work you've done to bring WordPerfect back.  I grew up with WP 5.1 for DOS and still keep 5.1+ setup in DOSBOX, but native 8c is a game-changer.

**Comment by emendelson** [comment]
Until this gets sorted out, download this .ALL file:

https://www.dropbox.com/s/i96hvmnjlqt4c4l/wp51gscr.all?dl=0

Install the driver named `Ghostscript Color PS (wpdos.org)` (or the Mono PS one if you prefer). It should support font size changes and prints the euro at WP character 4,72 (and, unlike the DOS version, the euro will appear in the editing screen).

This is the driver that I put together for the vDosWP and DOSBoxWP systems. It seems to work well in WP Unix also.

**Comment by abenson** [comment]
I appreciate it.  I'll be bringing this over to my DOS setups, as well.  Thanks!

**Comment by abenson** [comment]
I just tried again and now it works.

I feel dumb, I don't know what I did differently before.  I appreciate the help from both of you, in any case.

**Comment by emendelson** [comment]
And I wasted time by not checking this myself and jumping in with something irrelevant. You can of course use Tavis's ALL file with WPDOS also, and ignore the one that I linked.

**Comment by taviso** [comment]
It wasn't your fault @abenson! WordPerfect extracts the PRS file from the ALL file and stores it separately, so you likely installed the printer from a release where it was broken and then upgraded - not realizing you had the outdated PRS file!

At the moment I can only partially generate these .ALL files, some steps require manual changes, but I hope to fix that one day so tracking changes will be easier!

**Comment by taviso** [comment]
No problem :)

I'll close this issue, although I guess I should create an issue to look into a way to automatically update PRS files when the corresponding ALL file changes!

**Comment by abenson** [comment]
I found the root cause.  I was able to recreate my same issue on my other computer.

I'm building a [custom package for Void](https://github.com/abenson/void-packages/commit/2eca06ddc0bc34b9c18655b1578ddd0e302d1029.patch) just for ease a management and synchronization.  One of two things happened.

1. The tag/hash for v0.10 changed at some point and I had a broken version of it cached.
2. I really screwed up my package when messing with the update and somehow had a bad copy of the file cached.  

So in the end, I _thought_ I had v0.10 installed but the cached tarball did not have the right `ghostscr.all` file, so even deleting it would copy the broken one over.

Again, I appreciate the help!
